### DISCUSSION TITLE
Add soundswitch please (with a cherry on top)

### DISCUSSION THREAD

**Discussion by Silverwolf-3D** [ChrisTitusTech/winutil#25] [Ideas]
Hey<

small request to add soundswitch https://soundswitch.aaflalo.me/ they have a github so :)

thank you

**Comment by Carterpersall** [comment]
As this is a request for a feature within the script, it would be better placed in a Feature Request issue.
1. Click the issues tab
2. Click 'New issue'
3. Click 'Get started' on the 'Feature request' row
4. Fill out the template and submit the issue
